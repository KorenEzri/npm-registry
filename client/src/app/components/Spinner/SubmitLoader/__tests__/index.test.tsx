import * as React from 'react';
import { render } from '@testing-library/react';

import { SubmitLoader } from '..';


describe('<SubmitLoader  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<SubmitLoader />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
