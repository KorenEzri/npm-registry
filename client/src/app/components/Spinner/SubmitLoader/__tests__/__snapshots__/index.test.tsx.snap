// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitLoader  /> should match snapshot 1`] = `
.c1 {
  width: 60px;
  height: 60px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 .dot {
  width: 0.75em;
  height: 0.75em;
  border-radius: 50%;
  background: #fafafa;
  -webkit-animation-duration: 1.3s;
  animation-duration: 1.3s;
  -webkit-animation-iteration-count: infinite;
  animation-iteration-count: infinite;
}

.c0 .dot:nth-of-type(1) {
  -webkit-animation-name: loader_left;
  animation-name: loader_left;
}

.c0 .dot:nth-of-type(2) {
  -webkit-animation-name: loader_middle;
  animation-name: loader_middle;
}

.c0 .dot:nth-of-type(3) {
  -webkit-animation-name: loader_right;
  animation-name: loader_right;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="dot"
    />
    <div
      class="dot"
    />
    <div
      class="dot"
    />
  </div>
</div>
`;
